ui/src/pages/CreateAccount.js

```diff
@@ -1,65 +1,307 @@
-import { useState } from 'react';
-import { Button, Input, Label, Header } from 'semantic-ui-react';
+import { useEffect, useState } from 'react';
+import { Button, Input, Label } from 'semantic-ui-react';
+import "../App.css";
+import hiddenImage from "../icons/eye-close.png";
+import { appWindow, LogicalSize } from '@tauri-apps/api/window';
+import Loading from './Loading';
+import HyperLinkButton from '../components/HyperLinkButton';
 
 const MIN_PASSWORD_LENGTH = 8;
+const PASSWORD_TAB = 0;
+const SHOW_PHRASE_TAB = 1;
+const CONFIRM_PHRASE_TAB = 2;
+const FINAL_TAB = 3;
+const LOADING_TAB = 4;
 
-const CreateAccount = ({ recoveryPhrase, sendPassword, endInitialConnectionPhase }) => {
+const DEFAULT_WINDOW_SIZE = new LogicalSize(460, 500);
+const CONFIRM_PHRASE_WINDOW_SIZE = new LogicalSize(460, 900);
+
+const CreateAccount = (props) => {
```

Comment:
For readability purposes, I would just destructure all the props that this component receives as before